src/components/ContactsList.vue

```diff
@@ -157,6 +187,59 @@ export default {
 			}
 			return true
 		},
+		async newContact() {
+			const rev = new VCardTime()
```

Comment:
where's that code from? it appears moved. please delete the now unused method from the previous component

Reply:
its from here: https://github.com/nextcloud/contacts/blob/main/src/views/Contacts.vue#L38

i dont remember why i didnt remove it, but i dont think its because i forgot. I will test it asap and see